workflows/VGP-assembly-v2/Assembly-Hifi-only-VGP3/Assembly-Hifi-only-VGP3.ga

```diff
@@ -15,7 +15,7 @@
     ],
     "format-version": "0.1",
     "license": "CC-BY-4.0",
-    "release": "0.2.10",
+    "release": "0.2.11",
```

Comment:
```suggestion
    "release": "0.3.0",
```

I would keep patch version changes to anything that doesn't change inputs or outputs, like metadata changes